### DISCUSSION TITLE
Multiple Cropper at same time

### DISCUSSION THREAD

**Discussion by Mojtaba-T** [CropperBlazor/Cropper.Blazor#383] [Q&A] [answered]
Hi everyone,
I want to add multiple Cropper component at same time in the razor page.
I have some images which i want to crop. maybe one image, maybe 10 images, so it is dynamically.
Also i want to set dynamically @ref attribute so after finishing crop and submit, i can have cropped images for each image identically.

I have tried to use foreach for list of image bytes, turn to to src with method of GetImageUsingStreamingAsync but it is not working, i cant have final outputs.
thanks

**Comment by MaxymGorn** [comment, marked as answer]
@Mojtaba-T I strongly recommend not changing the @ref for the cropper yourself and using a different @ref for each cropper. Instead of src, use the replace functionality or recreate the cropper.

Also, let us know what type of project blazer you have, so that we can deal with other issues
